src/components/NotePage/NotePage.tsx

```diff
@@ -191,7 +198,12 @@ export default () => {
       }
       right={
         currentNote == null ? (
-          <div>No note selected</div>
+          <StyledNoteDetailNoNote>
+            <div>
+              <h1>Command(⌘) + N</h1>
```

Comment:
Isn't this really based on how you set your Hotkeys?

Reply:
It should be but we didn't implement shortcut settings yet. So it's okay for now.